2024-04-15 13:19:02 ERROR Elements not present after 0.5 seconds on page: https://capital.com/ru/tseny-na-syryo?t=fallers
2024-04-15 13:19:02 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 304, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x558ce2976863 <unknown>
#1 0x558ce266c8c6 <unknown>
#2 0x558ce26b7618 <unknown>
#3 0x558ce26b76d1 <unknown>
#4 0x558ce26fa744 <unknown>
#5 0x558ce26d95cd <unknown>
#6 0x558ce26f7c19 <unknown>
#7 0x558ce26d9343 <unknown>
#8 0x558ce26aa593 <unknown>
#9 0x558ce26aaf5e <unknown>
#10 0x558ce293a84b <unknown>
#11 0x558ce293e7a5 <unknown>
#12 0x558ce2928571 <unknown>
#13 0x558ce293f332 <unknown>
#14 0x558ce290d87f <unknown>
#15 0x558ce2965728 <unknown>
#16 0x558ce29658fb <unknown>
#17 0x558ce29759b4 <unknown>
#18 0x7f3ab9494ac3 <unknown>